app/views/departments/index.html.erb

```diff
@@ -16,9 +16,10 @@
       <tr>
         <td><%= department.name %></td>
         <td><%= department.description %></td>
-        <td><%= link_to 'Show', department, class: "btn btn-regular btn-secondary" %>
-        <%= link_to 'Edit', edit_department_path(department), class: "btn btn-outline-secondary" %>
-        <%= link_to 'Destroy', department, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-outline-danger" %>
+        <td><%= link_to 'Show', department, class: "btn btn-secondary" %>
+        <%= link_to 'Edit', edit_department_path(department), class: "btn btn-info" %>
+        <%= link_to 'Destroy', department, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-danger" %>
```

Comment:
use btn-outline-danger for delete/destroy buttons
and btn-regular for other simple buttons for purple color which goes with the theme